apps/web/src/app/tracks/_components/index.tsx

```diff
@@ -1,61 +1,114 @@
 import { getServerAuthSession, type Session } from '@repo/auth/server';
 import { prisma } from '@repo/db';
+import { EnrolledTrackSection } from './track-enrolled-section';
+import { PopularTrackSection } from './track-popular-section';
 
 export async function Tracks() {
   const session = await getServerAuthSession();
 
   const enrolledTracks = await getUserEnrolledTracks(session);
-  const tracks = await getTracks();
+  const tracks = await getPopularTracks();
 
   return (
-    <div>
-      <div>
-        {enrolledTracks?.tracks.map((et) => {
-          return (
-            <div key={et.id}>
-              <div>{et.title}</div>
-              <div>{et.description}</div>
-            </div>
-          );
-        })}
+    <div className="flex flex-col gap-8 py-8 md:gap-10 md:py-10">
+      <div className="container">
+        <h3 className="mb-1 text-2xl font-bold tracking-wide text-neutral-900/40 dark:text-white/40">
+          Welcome to
+        </h3>
+        <h1 className="mb-8 text-4xl font-bold tracking-tight text-neutral-900 dark:text-white">
+          TypeHero Tracks
+        </h1>
+        <p className=" max-w-[69ch] text-lg leading-10 text-neutral-600 dark:text-white/50">
+          TypeHero provides a curated collection of diverse coding challenges grouped into different
+          tracks, offering a dynamic learning path for developers to enhance their skills.
+        </p>
       </div>
-      {tracks?.map((t) => {
-        return (
-          <div key={t.id}>
-            <div>{t.title}</div>
-            <div>{t.description}</div>
-          </div>
-        );
-      })}
+      {session?.user && (
+        <EnrolledTrackSection
+          key="enrolled-track-section"
+          title="Your Tracks"
+          tracks={enrolledTracks}
+        />
+      )}
+      <PopularTrackSection
```

Comment:
lets just list them out vs carousel. we're launching with small number of tracks

Reply:
oki! since the carousel is generalised. you can just plop it the children at any time you need it in future. (which u will as [you've said here trash](https://github.com/bautistaaa/typehero/pull/451#issuecomment-1694538323))